src/doc/src/reference/publishing.md

```diff
@@ -240,8 +240,23 @@ the “Grant Access” button next to its name:
 
 ![Authentication Access Control](../images/auth-level-acl.png)
 
+#### could not find the github team
+
+Or, when trying to add a GitHub team as crate owner, you may see an error like:
+
+```text
+error: failed to invite owners to crate <crate_name>: api errors (status 200 OK): could not find the github team org/repo
+```
+In that case, you should go to [the GitHub Application settings page] and
+check if crates.io is listed in the `Authorized OAuth Apps` tab.
+If it isn't, you should go to <https://crates.io/login> and authorize it.
```

Comment:
https://crates.io/login doesn't seem to be a valid URL.

Reply:
Oh, the error message on crates.io is outdated, will fix it.

Reply:
The message was updated by rust-lang/crates.io#3146.